src/TodoItem.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoItem = props => (
```

Comment:
```suggestion
const TodoItem = ({ todo }) => (
```
Делай деструктуризацию props